website/src/blogs/data-migration-within-kubernetes.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Data Migration Within Kubernetes Clusters
+author: Sai Chaithanya
+author_info: A developer whos is always eager to learn, loves algorithms, maths, Kubernetes, and programming, Passionate towards Data Science. Enjoys playing kabaddi and traveling.
+excerpt: In this blog, we'll talk about migrating data within Kubernetes from one node to another without any downtime of the application.
+tags: Kubernetes
+date: 02-07-2020
+---
+
+In large scale environments, storage is one of the hard things to manage, and it will be the most crucial component as it has DATA with it. OpenEBS, leading open source Cloud Native Storage, makes managing storage easier in Kubernetes environments. Mayadata, the company behind the OpenEBS project, has the vision of achieving data agility by transforming Kubernetes as a data plane. cStor is one of the storage engines of OpenEBS. 
```

Comment:
```suggestion
In large scale environments, storage is one of the hard things to manage, and it will be the most crucial component as it has DATA with it. OpenEBS, leading open source Cloud Native Storage, makes managing storage easier in Kubernetes environments. MayaData, the company behind the OpenEBS project, has the vision of achieving data agility by transforming Kubernetes as a data plane. cStor is one of the storage engines of OpenEBS. 
```